amrit-local-setup/README.md

```
@@ -1,100 +1,642 @@
-3. Update the data path in the appropriate script:
-   - Line 10 in `loaddummydata.sh` (Linux/MacOS)
-   - Line 10 in `loaddummydata.bat` (Windows)
+**Build and run schema migrations:**
+```bash
+mvn clean install -DENV_VAR=local
+mvn spring-boot:run -DENV_VAR=local
+```
 
-#### Execute Data Load
+**Monitor the logs** for migration progress. You should see:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)